include/xrpl/resource/Charge.h

```diff
@@ -55,6 +55,8 @@ class Charge
     operator==(Charge const&) const;
     bool
     operator!=(Charge const&) const;
+    bool
+    operator>=(Charge const&) const;
```

Comment:
Would it make more sense to implement the spaceship operator (`<=>`) instead of adding specific comparisons one at a time?